src/Frontend.elm

```diff
@@ -746,30 +746,7 @@ eventsToEvent2Helper state =
     Maybe.Extra.toList state.previousEvent
         ++ state.rest
         |> List.reverse
-        |> List.foldl
-            (\event a ->
-                { previousEvent = Just event
-                , list =
-                    case a.previousEvent of
-                        Just { time } ->
-                            let
-                                delta : Int
-                                delta =
-                                    event.time - time
-                            in
-                            if delta > 0 then
-                                event.eventType :: a.list
```

Comment:
I've simplified this one down partially manually, but it's suspicious, double check it

Reply:
Yeah, this is weird. I obviously was intending to do something here at some point but right now it's a big noop. Let's just merge all this in for now and do some tests later once all the changes are done.